tests/MiniPdf.Benchmark/generate_office_pdfs_pptx.py

```
@@ -0,0 +1,86 @@
+                continue
+
+            print(f"  Converting {pptx_path.name} ...", end=" ", flush=True)
+            presentation = None
+            try:
+                presentation = powerpoint.Presentations.Open(
+                    os.path.abspath(str(pptx_path)), -1, 0, 0
+                )
+                presentation.SaveAs(os.path.abspath(pdf_path), 32)
+                passed += 1
+                print("OK")
+            except Exception as error:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)